bin/phpactor.php

```diff
@@ -1,38 +1,37 @@
 <?php
+use Phpactor\Console\Application;
+
+use Symfony\Component\Debug\Debug;
 
 use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Output\ConsoleOutput;
 
-$autoloadFile = __DIR__ . '/../vendor/autoload.php';
-if (!file_exists($autoloadFile)) {
+foreach ([__DIR__ . '/../../autoload.php', __DIR__ . '/../vendor/autoload.php', __DIR__ . '/vendor/autoload.php'] as $file) {
+    if (file_exists($file)) {
+        define('PHPACTOR_COMPOSER_INSTALL', $file);
+        break;
+    }
+}
+
+if (!defined('PHPACTOR_COMPOSER_INSTALL')) {
     echo sprintf(
         'Phpactor dependencies not installed. Run `composer install` (https://getcomposer.org) in "%s"' . PHP_EOL,
         realpath(__DIR__ . '/..')
     );
     exit(255);
 }
 
-require_once($autoloadFile);
-
-$minVersion = '7.0.0';
-
-if (version_compare(PHP_VERSION, $minVersion) < 0) {
-    echo 'Phpactor requires at least PHP ' . $minVersion;
-    exit(255);
-}
-
-use Phpactor\Console\Application;
-use Symfony\Component\Console\Output\ConsoleOutput;
-use Symfony\Component\Debug\Debug;
+require_once PHPACTOR_COMPOSER_INSTALL;
 
 Debug::enable();
 
 $application = new Application();
 $output = new ConsoleOutput();
 
-try { 
+try {
     $application->initialize();
     $application->run(null, $output);
-} catch (\Exception $e) {
+} catch (Exception $e) {
```

Comment:
`\` not needed here as we're on a namespace-less file, so `Exception` is already mapping to `\Exception`. Can revert if needed.